.github/workflows/lint-test.yaml

```diff
@@ -0,0 +1,81 @@
+name: Lint and Test Charts
+
+on: pull_request
+
+jobs:
+  lint-test:
```

Comment:
This does a lot more than linting, so I think another name would be good, but I'm not sure what to suggest.  `lint-and-deploy` maybe?

Reply:
I agree with the renaming, I have updated the workflow name.